docs/meetings/team/2023_01_30.md

```diff
@@ -0,0 +1,69 @@
+# 🚀 Weekly Team Meeting Minutes
+
+**Start Time** 10:00
+
+**End Time** 10:48
+
+**Location** Discord
+
+**Date** January 30, 2023
+
+## 👋 Attendence
+
+1. Veronica
+2. Logan
+3. Edouard
+4. Matt
+
+***Attendence Comments***
+
+- Logan won't be in class this week due to medical reasons
+
+## 🃏 Joke of the Day
+
+What did one tide pool say to the other tide pool? Show me your mussels!
+
+## 📢 Announcements
+
+- Week 18 GitStats due this week on Friday
+- there was a new deliverable, but it's no longer on Canvas
+- Rhubarb developer replied again! Check Discord for a screenshot of the message
+
+## 📅 Last Week (Restrospective)
+
+- Ed built out tests for the animation factory that run well locally, but when they run in GitHub actions, they fail.
+- Last week went well, no comment
+- Matt was looking into C++ to implement the Rhubarb hack
+
+## 📋 This Week
+
+### Assets
+
+- We will need to make a request to the artist for different positions
+  - Nazim asked that we be as specific as possible
+  - Ideas for body language
+    1) hand on hip, other hand down
+    2) one hand up to explain
+    3) two hands up to explain
+- Once we receive the art, we'll need to export the assets for our file setup
+
+### Issues this week
+
+- Find a paper about body language and speaking positions so that we have a good set of positions to request
+- Veronica will reply to the Rhubarb developer (1 is true, 2 is usually false)
```

Comment:
Just rereading the email, it does say no pauses, and there are times where we do have some pause in the speech, not sure if that would cause issues

Reply:
Yeah, I'll mention there is punctuation which may result in pauses in speech for TTS.